src/initramfs-tools/loopback-file-system-support.patch

```diff
@@ -13,11 +13,29 @@ And added NFS support! Full patch below...
 live well,
   vagrant
 
+---
+ init              | 12 ++++++++++++
```

Comment:
this is to adapt the loopback file system support patch to v0.133. the main changes are due to the introduction of shellcheck.